dashboard/ui.R

```diff
@@ -50,10 +50,12 @@ fluidPage(
                                  tags$br(), 
                                  htmlOutput("iri"),
                                  htmlOutput("icpac"),
-                                 #img(src="icpac_rainfall_20201222_JFM2021.png", align = "center", width=600, height=600),                                 
-                                 htmlOutput("chc"),
+                                 #img(src="icpac_rainfall_20201222_JFM2021.png", align = "center", width=600, height=600),
+                                 img(src="eth_nma_precipitation_202101_Belg2021.png", align = "center", width=600, height=400),
```

Comment:
this does not display correctly. You have to save the PNG in the www folder.

Reply:
Oops I forgot to add the png to Git🤦‍♀️